versionedRouter.js

```diff
@@ -99,9 +100,35 @@ const functionsEnabled = () => {
   return areFunctionsEnabled === 1;
 };
 
-async function handleCdkV2(destName, parsedEvent, flowType) {
+// eslint-disable-next-line no-unused-vars
+function getCommonMetadata(ctx) {
+  // TODO: Parse information such as
+  // cluster, namespace, etc information
+  // from the request
+  return {
+    namespace: "Unknown",
+    cluster: "Unknown"
+  };
+}
+
+async function handleCdkV2(destType, parsedEvent, flowType) {
+  const getReqMetadata = () => {
+    try {
+      return {
+        flowType,
+        destType,
+        destinationId: parsedEvent.destination.ID,
```

Comment:
as we discussed change to parsedEvent.destination?.ID

Reply:
Addressed.